src/dune_engine/build_system.ml

```diff
@@ -367,7 +367,7 @@ let get_dir_triage t ~dir =
     Memo.Build.return
     @@ Dir_triage.Known
          (Non_build
-            (match Path.readdir_unsorted dir with
+            (match Path.Untracked.readdir_unsorted dir with
```

Comment:
Ouch, looks like we need to add `let readdir_unsorted = `Use_fs_memo_dir_contents_instead` to `dune_engine/import.ml` so that we were forced to write `Untracked` here.

Reply:
Oh, I see you did that, nice :)

Reply:
Indeed (that's what I did)